content/docs/how-to-contribute.md

```diff
@@ -145,6 +145,13 @@ However, there are still some styles that the linter cannot pick up. If you are
 
 You may be interested in watching [this short video](https://www.youtube.com/watch?v=wUpPsEcGsg8) (26 mins) which gives an introduction on how to contribute to React.
 
+#### Available topics in this video directly linked:
```

Comment:
How do you feel about a title more like "Video highlights"

Reply:
just perfect, I'll adjust and update.